<!doctype html>
<html>
  <head>
    <title>LGSF Dashboard</title>
    <link rel="stylesheet"  href="/lgsf-dashboard/assets/css/styles.css">
    <link rel="shortcut icon" href="https://democracyclub.org.uk/static/images/logo_icon.png">
  </head>
  <body>
    <div class="ds-page">
      <a class="ds-skip-link" href="#main">skip to content</a>

      <header class="ds-header">
        <a class="ds-skip-link" href="#main">skip to content</a>
        <a class="ds-logo" href="/lgsf-dashboard/">
          <img src="https://DemocracyClub.github.io/design-system/images/logo_icon.svg" alt="" />
          <span>LGSF Logbooks</span>
        </a>

      </header>


      <main id="main" tabindex="-1" class="ds-stack">

        
<style>

    pre {
        height:400px;
        overflow-x: scroll;
        width:100%
    }
</style>




  


  <h2 id="2022-04-26-12-08">2022-04-26</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>144 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-26 12:08:47.091544</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-26 12:11:11.467339</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>0</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd></dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:08:47] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:230
           Getting all files in KEC...                               base.py:182
[12:08:48] Getting all files in KEC/json...                          base.py:182
           ...found 49 files in KEC/json                             base.py:198
           Getting all files in KEC/raw...                           base.py:182
           ...found 49 files in KEC/raw                              base.py:198
           ...found 99 files in KEC                                  base.py:198
           Deleting batch no. 1 consisting of 99 files               base.py:207
[12:08:49] ...data deleted.                                          base.py:237
           Scraping from                                              base.py:42
           https://www.rbkc.gov.uk/committees/Councillors.aspx                  
[12:08:50] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:08:52] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/448/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:08:53] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/479/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:08:54] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/450/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:08:55] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/262/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:00] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/404/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:01] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/458/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:02] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/392/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:03] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/226/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:20] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/283/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:26] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/456/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:27] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/422/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:28] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/297/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:31] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/453/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:33] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/446/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:34] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/383/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:35] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/221/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:42] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/447/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:43] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/313/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:45] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/227/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:48] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/444/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:49] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/270/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:52] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/440/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:53] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/443/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:54] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/459/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:55] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/449/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:56] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/268/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:09:59] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/303/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:05] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/239/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:20] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/441/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:21] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/314/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:23] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/445/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:24] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/451/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:26] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/285/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:28] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/310/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:30] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/395/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:32] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/461/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:33] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/308/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:35] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/454/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:36] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/455/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:37] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/410/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:39] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/452/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:40] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/460/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:41] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/323/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:42] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/442/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:43] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/243/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:50] Committing batch 1 consisting of 92 files                 base.py:265
[12:10:51] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/322/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:10:55] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/281/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:11:06] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/457/ScreenMode/           
           Alphabetical/Default.aspx                                            
[12:11:09] Committing batch 2 consisting of 6 files                  base.py:265
[12:11:11] Finished attempting to scrape: KEC                        base.py:315
</pre>
  

  


  <h2 id="2022-04-25-13-33">2022-04-25</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>135 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-25 13:33:00.695654</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-25 13:35:16.329413</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>0</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd></dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:33:00] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:230
[13:33:01] Getting all files in KEC...                               base.py:182
           Getting all files in KEC/json...                          base.py:182
           ...found 49 files in KEC/json                             base.py:198
           Getting all files in KEC/raw...                           base.py:182
[13:33:02] ...found 49 files in KEC/raw                              base.py:198
           ...found 99 files in KEC                                  base.py:198
           Deleting batch no. 1 consisting of 99 files               base.py:207
[13:33:03] ...data deleted.                                          base.py:237
           Scraping from                                              base.py:42
           https://www.rbkc.gov.uk/committees/Councillors.aspx                  
[13:33:04] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:05] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/448/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:07] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/479/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:08] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/450/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:10] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/262/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:22] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/404/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:24] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/458/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:25] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/392/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:26] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/226/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:34] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/283/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:37] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/456/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:38] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/422/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:39] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/297/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:43] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/453/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:44] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/446/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:46] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/383/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:47] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/221/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:54] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/447/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:55] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/313/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:33:57] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/227/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:03] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/444/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:06] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/270/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:13] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/440/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:14] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/443/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:15] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/459/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:16] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/449/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:17] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/268/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:20] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/303/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:22] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/239/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:29] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/441/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:30] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/314/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:33] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/445/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:34] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/451/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:35] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/285/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:39] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/310/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:41] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/395/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:42] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/461/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:43] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/308/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:46] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/454/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:48] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/455/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:50] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/410/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:54] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/452/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:57] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/460/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:34:58] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/323/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:35:00] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/442/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:35:01] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/243/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:35:05] Committing batch 1 consisting of 92 files                 base.py:265
[13:35:06] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/322/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:35:09] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/281/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:35:12] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/457/ScreenMode/           
           Alphabetical/Default.aspx                                            
[13:35:14] Committing batch 2 consisting of 6 files                  base.py:265
[13:35:16] Finished attempting to scrape: KEC                        base.py:315
</pre>
  

  


  <h2 id="2022-04-24-20-32">2022-04-24</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>122 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-24 20:32:54.153087</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-24 20:34:56.630163</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>0</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd></dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[20:32:54] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:230
[20:32:55] Getting all files in KEC...                               base.py:182
           ...found 1 files in KEC                                   base.py:198
           Deleting batch no. 1 consisting of 1 files                base.py:207
[20:32:56] ...data deleted.                                          base.py:237
           Scraping from                                              base.py:42
           https://www.rbkc.gov.uk/committees/Councillors.aspx                  
[20:32:57] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:32:58] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/448/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:00] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/479/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:01] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/450/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:02] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/262/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:09] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/404/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:11] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/458/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:12] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/392/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:14] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/226/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:21] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/283/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:25] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/456/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:26] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/422/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:27] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/297/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:31] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/453/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:32] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/446/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:34] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/383/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:35] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/221/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:42] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/447/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:44] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/313/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:46] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/227/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:49] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/444/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:50] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/270/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:54] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/440/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:55] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/443/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:57] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/459/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:33:58] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/449/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:00] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/268/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:03] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/303/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:05] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/239/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:12] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/441/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:14] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/314/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:17] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/445/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:18] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/451/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:19] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/285/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:22] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/310/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:24] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/395/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:26] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/461/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:27] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/308/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:29] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/454/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:30] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/455/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:32] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/410/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:33] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/452/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:35] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/460/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:36] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/323/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:38] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/442/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:39] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/243/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:45] Committing batch 1 consisting of 92 files                 base.py:265
[20:34:46] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/322/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:49] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/281/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:53] Scraping from https://www.rbkc.gov.uk/committees/Councillo base.py:42
           rs/tabid/62/ctl/ViewCMIS_Person/mid/384/id/457/ScreenMode/           
           Alphabetical/Default.aspx                                            
[20:34:55] Committing batch 2 consisting of 6 files                  base.py:265
[20:34:56] Finished attempting to scrape: KEC                        base.py:315
</pre>
  

  


  <h2 id="2022-04-23-13-47">2022-04-23</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-23 13:47:51.878864</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-23 13:47:55.714136</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fd83dcc9040>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd83dcc9040>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 123, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd83dcc9040>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:47:51] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:228
[13:47:52] Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[13:47:53] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[13:47:55] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fd83dcc9040>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-22-11-21">2022-04-22</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-22 11:21:54.412194</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-22 11:21:58.286398</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f942dcecbe0>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f942dcecbe0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 123, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f942dcecbe0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:21:54] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:228
[11:21:55] Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[11:21:56] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:21:57] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
[11:21:58] HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f942dcecbe0>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-21-16-14">2022-04-21</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-21 16:14:04.910136</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-21 16:14:09.090632</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fbad158b730>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbad158b730>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbad158b730>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[16:14:04] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
[16:14:05] Deleting existing data...                                 base.py:228
           Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[16:14:06] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[16:14:08] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fbad158b730>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
[16:14:09] Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-20-13-10">2022-04-20</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>21 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-20 13:10:21.762657</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-20 13:10:43.147104</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fd6f05f65b0>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd6f05f65b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd6f05f65b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:10:21] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:228
[13:10:22] Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[13:10:23] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[13:10:42] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fd6f05f65b0>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
[13:10:43] Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-19-11-32">2022-04-19</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>39 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-19 11:32:33.471070</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-19 11:33:13.222811</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f05b2e59eb0>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f05b2e59eb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f05b2e59eb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[11:32:33] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:228
[11:32:34] Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[11:32:35] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[11:33:12] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f05b2e59eb0>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
[11:33:13] Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-18-16-03">2022-04-18</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>6 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-18 16:03:02.293511</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-18 16:03:08.924978</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7ff3837eb700>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff3837eb700>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff3837eb700>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[16:03:02] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:228
[16:03:03] Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[16:03:04] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[16:03:08] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7ff3837eb700>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-17-15-14">2022-04-17</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>5 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-17 15:14:48.746069</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-17 15:14:54.682852</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f464e697b50>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f464e697b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f464e697b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[15:14:48] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:228
[15:14:49] Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[15:14:50] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[15:14:54] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f464e697b50>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-16-13-01">2022-04-16</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-16 13:01:18.285301</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-16 13:01:22.982049</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fb9dfc4cf70>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb9dfc4cf70>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb9dfc4cf70>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:01:18] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:228
           Getting all files in KEC...                               base.py:180
[13:01:19] ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[13:01:20] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[13:01:22] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fb9dfc4cf70>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-15-13-49">2022-04-15</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>6 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-15 13:49:29.412264</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-15 13:49:36.076964</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fa5932ad910>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa5932ad910>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa5932ad910>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:49:29] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
           Deleting existing data...                                 base.py:228
[13:49:30] Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[13:49:31] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[13:49:35] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fa5932ad910>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
[13:49:36] Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-14-14-45">2022-04-14</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>6 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-14 14:45:57.941014</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-14 14:46:03.945771</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7ff1cbe35910>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff1cbe35910>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff1cbe35910>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:45:57] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:27
[14:45:58] Deleting existing data...                                 base.py:228
           Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[14:45:59] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[14:46:03] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:36
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7ff1cbe35910>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-13-12-09">2022-04-13</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>21 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-13 12:09:13.685155</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-13 12:09:35.088318</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f1e3cd926d0>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1e3cd926d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 31, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1e3cd926d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:09:13] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:26
           Deleting existing data...                                 base.py:228
[12:09:14] Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[12:09:15] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[12:09:34] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:35
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f1e3cd926d0>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
[12:09:35] Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-12-13-24">2022-04-12</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>14 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-12 13:24:32.805912</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-12 13:24:47.100126</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f46a5c10d30>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f46a5c10d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 31, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f46a5c10d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:24:32] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:26
           Deleting existing data...                                 base.py:228
[13:24:33] Getting all files in KEC...                               base.py:180
           ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[13:24:34] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[13:24:46] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:35
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7f46a5c10d30>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
[13:24:47] Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  


  <h2 id="2022-04-11-13-23">2022-04-11</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>8 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2022-04-11 13:23:18.199765</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2022-04-11 13:23:26.472961</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/opt/python/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/opt/python/urllib3/util/connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/var/lang/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/opt/python/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/opt/python/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/opt/python/urllib3/connection.py", line 358, in connect
    conn = self._new_conn()
  File "/opt/python/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7fd9aa5b1790>: Failed to establish a new connection: [Errno -2] Name or service not known

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/opt/python/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/opt/python/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd9aa5b1790>: Failed to establish a new connection: [Errno -2] Name or service not known'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 31, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 48, in run
    councillor = self.get_single_councillor(councillor_html)
  File "scrapers/KEC-kensington-and-chelsea/councillors.py", line 16, in get_single_councillor
    soup = self.get_page(url)
  File "/var/task/lgsf/councillors/scrapers.py", line 115, in get_page
    page = self.get(url).text
  File "/var/task/lgsf/scrapers/base.py", line 46, in get
    response = requests.get(url, headers=headers, verify=verify)
  File "/opt/python/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/python/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/python/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/python/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/opt/python/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.rbkc.gov.ukhttps', port=443): Max retries exceeded with url: //www.rbkc.gov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid/384/id/421/ScreenMode/Alphabetical/Default.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fd9aa5b1790>: Failed to establish a new connection: [Errno -2] Name or service not known'))
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:23:18] Fetching Scraper for: KEC                              handlers.py:23
           Begin attempting to scrape: KEC                        handlers.py:26
           Deleting existing data...                                 base.py:228
           Getting all files in KEC...                               base.py:180
[13:23:19] ...found 1 files in KEC                                   base.py:196
           Deleting batch no. 1 consisting of 1 files                base.py:205
[13:23:20] ...data deleted.                                          base.py:235
           Scraping from https://www.rbkc.gov.uk/council-and-democrac base.py:40
           y/councillors-and-committees/councillors/cllrs-ward/counci           
           llors-and-their-wards                                                
[13:23:26] Scraping from https://www.rbkc.gov.ukhttps://www.rbkc.gov. base.py:40
           uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Person/mid           
           /384/id/421/ScreenMode/Alphabetical/Default.aspx                     
           HTTPSConnectionPool(host='www.rbkc.gov.ukhttps',       handlers.py:35
           port=443): Max retries exceeded with url: //www.rbkc.g               
           ov.uk/committees/Councillors/tabid/62/ctl/ViewCMIS_Per               
           son/mid/384/id/421/ScreenMode/Alphabetical/Default.asp               
           x (Caused by NewConnectionError('<urllib3.connection.H               
           TTPSConnection object at 0x7fd9aa5b1790>: Failed to                  
           establish a new connection: [Errno -2] Name or service               
           not known'))                                                         
           Finished attempting to scrape: KEC                        base.py:313
</pre>
  

  

  

  

  


      </main>
      <footer class="ds-footer">...</footer>
    </div>
  </body>
